README.md

```diff
@@ -0,0 +1,32 @@
+# lockgoose
+
+## Install
+
+```bash
+npm i lockgoose
+```
+
+## Usage
+
+```javascript
+const mongoose = require('mongoose');
+const lockgoose = require('lockgoose')(mongoose);
```

Comment:
minor: you could avoid this with a peer dependency https://nodejs.org/en/blog/npm/peer-dependencies/

Reply:
original idea was that different connections could be passed through but on reflection that would need more initialisation than would fall under "basic implementation"